.goreleaser.yml

```diff
@@ -202,4 +202,4 @@ snapshot:
   name_template: "{{ .Tag }}-next"
 
 release:
-  draft: true
+  prerelease: true
```

Comment:
https://github.com/softprops/action-gh-release/issues/215

TODO: use back draft after the issue is fixed.